pkg/vfs/directory.go

```diff
@@ -29,10 +29,10 @@ type DirDoc struct {
 
 	CreatedAt time.Time `json:"created_at"`
 	UpdatedAt time.Time `json:"updated_at"`
-	Tags      []string  `json:"tags"`
+	Tags      []string  `json:"tags,omitempty"`
 
 	// Directory path on VFS
-	Fullpath string `json:"path"`
+	Fullpath string `json:"path,omitempty"`
```

Comment:
In which case we can have the Fullpath empty?

Reply:
`DirDoc` is used as the base of `DirOrFileDoc`, this is the reason why I added an omitempty. I'll add a comment to make it clear.